cinema/serializers.py

```diff
@@ -68,13 +78,60 @@ class Meta:
             "movie_title",
             "cinema_hall_name",
             "cinema_hall_capacity",
+            "tickets_available",
         )
 
 
+class TicketSerializer(serializers.ModelSerializer):
+    def validate(self, attrs):
+        data = super(TicketSerializer, self).validate(attrs=attrs)
+        Ticket.validate_ticket(
+            attrs["row"], attrs["seat"], attrs["movie_session"]
+        )
```

Comment:
The method `Ticket.validate_ticket` is called here, but it's not clear if this method exists in the `Ticket` model. Ensure that `validate_ticket` is implemented in the `Ticket` model to avoid runtime errors.